.storybook/test-data.js

```diff
@@ -1227,14 +1228,16 @@ const state = {
         metadata: {
           EIPS: { 1559: true },
           status: NetworkStatus.Available,
-        }
-      }, {
+        },
+      },
+      {
         id: 'test-networkConfigurationId-2',
         rpcUrl: 'http://localhost:8545',
         chainId: '0x539',
         ticker: 'ETH',
         nickname: 'Localhost 8545',
-      }),
+      },
+    ),
     accountTokens: {
```

Comment:
weird lint diff. Did something change with our config?

Reply:
Dnt think config has changed, will revert this change 🙏